registry/access_control/rbac/auth.py

```diff
@@ -115,14 +115,14 @@ def _decode_token(self, token: str) -> Mapping:
                                 'RS256'], audience=config.RBAC_API_AUDIENCE)
             return decode
         except ExpiredSignatureError as e:
-            logging.debug(f'The token signature has expired: {e}')
+            logging.error(f'The token signature has expired: {e}')
             raise InvalidAuthorization('The token signature has expired')
         except PyJWKError as e:
-            logging.debug(f'Invalid token: {e}')
+            logging.error(f'Invalid token: {e}')
             raise InvalidAuthorization('The token is invalid')
         except Exception as e:
-            logging.debug(f'Unexpected error: {e}')
-            raise InvalidAuthorization('Unable to decode token, error: {e}')
+            logging.error(f'Unexpected error: {e}')
+            raise InvalidAuthorization(f'Unable to decode token, error: {e}')
```

Comment:
>raise InvalidAuthorization(f'Unable to decode token, error: {e}') [](http://example.com/codeflow?start=12&length=65)

Unable to decode token indicates server error, should return 500 internal server error instead.